prow/flagutil/github.go

```diff
@@ -58,6 +61,8 @@ func (o *GitHubOptions) addFlags(wantDefaultGitHubTokenPath bool, fs *flag.FlagS
 	}
 	fs.StringVar(&o.TokenPath, "github-token-path", defaultGitHubTokenPath, "Path to the file containing the GitHub OAuth secret.")
 	fs.StringVar(&o.deprecatedTokenFile, "github-token-file", "", "DEPRECATED: use -github-token-path instead.  -github-token-file may be removed anytime after 2019-01-01.")
+	fs.StringVar(&o.gitHost, "git-host", "", "Host to contact for git operations.")
```

Comment:
This field lives in GitHubOptions while it is named as `git-host` instead `github-host`, I feel this is a little bit confusing.

If we like to decouple with github, should this live in its own options struct?